### PR TITLE
Add docker support for tools

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

File: docker/release/Dockerfile.tools.release [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+VOLUME /alephium-home/.alephium
+
+USER nobody
+
+ENV JAVA_NET_OPTS="-Djava.net.preferIPv4Stack=true"
+ENV JAVA_MEM_OPTS="-Xms512M -XX:+UseStringDeduplication -XX:+OptimizeStringConcat -XX:+UseNUMA"
+ENV JAVA_GC_OPTS="-XX:+UseShenandoahGC -XX:ShenandoahGCHeuristics=static -XX:+UnlockExperimentalVMOptions -XX:ShenandoahGuaranteedGCInterval=16000 -XX:ShenandoahUncommitDelay=60000 -XX:MaxGCPauseMillis=1 -XX:+ParallelRefProcEnabled -XX:-UseGCOverheadLimit -XX:+DisableExplicitGC"
+ENV JAVA_EXTRA_OPTS=""
+
+ENTRYPOINT /entrypoint.sh
```
Comment (polarker) [inline docker/release/Dockerfile.tools.release:22]: should be `entrypoint-tools.sh`?

Comment (polarker) [reply docker/release/Dockerfile.tools.release:22]: Nope. It's renamed already.

File: docker/release/entrypoint-tools.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+echo "Starting java ${JAVA_NET_OPTS} ${JAVA_MEM_OPTS} ${JAVA_GC_OPTS} ${JAVA_EXTRA_OPTS} -cp /alephium-tools.jar $@"
+exec java ${JAVA_NET_OPTS} ${JAVA_MEM_OPTS} ${JAVA_GC_OPTS} ${JAVA_EXTRA_OPTS} -cp /alephium.jar $@
```
Comment (polarker) [inline docker/release/entrypoint-tools.sh:4]: Should it be `alephium-tools.jar`?

Comment (h0ngcha0) [reply docker/release/entrypoint-tools.sh]: Yes if we want this image to be specific about prune storage. We can also add more commands to this image in the future. 

Right now for pruning we can run something like this:
```
docker run -it -v /mnt/disks/alephium-data:/alephium-home/.alephium  alephium/alephium-tools:2.6.1 org.alephium.tools.PruneStorage
```

**Comment by snoby** [comment]
hum.... i tried duplicating this and i don't know java that well but...
```
nobody@783971f17e4c:/$ java  -cp /alephium-tools.jar
Usage: java [options] <mainclass> [args...]
```
and then when i tried giving it the full path...
```
nobody@783971f17e4c:/$ java  -cp /alephium-tools.jar org.alephium.tools.PruneStorage
Error: Could not find or load main class org.alephium.tools.PruneStorage
Caused by: java.lang.ClassNotFoundException: org.alephium.tools.PruneStorage
```

**Pull request merged**